Documentation/guides/DotNet5.md

```diff
@@ -120,4 +120,58 @@ The following instructions can be used for early preview testing.
 dotnet publish -t:Install *.csproj
 ```
 
-[0]:  https://github.com/dotnet/installer#installers-and-binaries
+## Package Versioning Scheme
+
+Our NuGet packages are versioned using [Semver 2.0.0][2].
+
+This is the scheme: `OsMajor.OsMinor.InternalRelease[-prereleaseX]+sha.1b2c3d4`.
```

Comment:
Suggest using `OS-Major` and `OS-Minor` instead of `OsMajor` and `OsMinor`; I find the latter ugly.